### PR TITLE
DS-351 Configure services.json for predictiva

### PR DESCRIPTION
**Pull request by agomezandrades**

### CODE REVIEW

File: opusMT-servers-UH.json [hunk 1]
```diff
@@ -1,83 +1,83 @@
...
+    },
+    "192.168.1.18:20000": {
+        "source-languages": "en",
+        "target-languages": "fi"
+    },
+    "192.168.1.18:20001": {
+        "source-languages": "fi",
+        "target-languages": "en"
+    },
+    "192.168.1.18:20005": {
```
Comment (evicentepred) [inline opusMT-servers-UH.json:18]: Just curiosity, is there any reason to change this line (previously fi-de) and not any other?

**Review by evicentepred — APPROVED** [review]
Ok, if with these changes you have successfully run it, perfect.

File: opusMT-servers-UH.json (hunk 1 above)
Comment (agomezandrades) [reply opusMT-servers-UH.json:18]: no XDDDD It was a random selection, not sure if this is needed

Comment (kir4h) [reply opusMT-servers-UH.json:18]: This seems to point to a specific server instance in charge of that specific translation. But this is just a wild guessing, I haven't even read what the repo is about. I'd say there is no need.

I guess `services.json` defines what to instance and this file defines a list of external instances you cann connect to?

Comment (agomezandrades) [reply opusMT-servers-UH.json:18]: Yes, I think services.json does what you say and it is enough to make it work

**Pull request merged**